### DISCUSSION TITLE
https://docs.mapbox.com/mapbox-gl-js/example/mapbox-gl-draw/

### DISCUSSION THREAD

**Discussion by SanjanaTailor** [rnmapbox/maps#1953] [General]
Hello, 

https://docs.mapbox.com/mapbox-gl-js/example/mapbox-gl-draw/

can this feature be available in React native project ? and how can we implement for React native ?

**Comment by bllanos** [comment]
We have implemented a React Native library for drawing points, polylines, and polygons (currently without holes) here: https://github.com/newfarms/react-native-mapbox-geometry-editor

As one of the authors, I can't comment on how good it is, but hopefully it helps.
